client/reader/conversations/controller.js

```diff
@@ -60,6 +60,7 @@ export function conversationsA8c( context, next ) {
 			key="conversations"
 			title="Conversations @ Automattic"
 			streamKey={ streamKey }
+			store={ { id: streamKey } }
```

Comment:
Why was this line added?

Reply:
This line fixes [this check on the `isInternal`](https://github.com/Automattic/wp-calypso/pull/97353/files#diff-d126b3f097f7d6ab26830253597fb19bc7032fd1f51eda2bf5b3194b83d56251R62) so it shows the "Welcome to A8C Conversations" intro. That `isInternal` var is set [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/conversations/stream.js#L13) and needs this prop.